src/platforms/php/guides/symfony/performance/pm-integrations.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: Performance Integrations
+description: "Learn about the the integrations Sentry provides for performance monitoring and how to configure them."
+sidebar_order: 500
+---
+
+The Symfony SDK offers integrations that support distributed tracing, including database queries, Twig template rendering, and cache pools. These integrations are enabled by default.
+
+To use them, you need to use the latest version of the SDK and add the specific integration as illustrated below:
+
+```yaml {filename:config/packages/sentry.yaml}
+sentry:
+  tracing:
+    enabled: true
+    dbal: # DB queries
+      enabled: true
+    cache:
+      enabled: true
+    twig: # templating engine
+      enabled: true
+```
+
+### Dbal
+
+This integration supports tracking database queries as Spans. Enabling this option will create Spans for your queries.
+
+It is enabled by default. To disable it, set `enabled` to `false`.
```

Comment:
Since there are two `enabled` keys, I suggest to either remove `sentry.tracing.enabled` (which is the global toggle and has no meaning in the context of this specific part of the documentation) or to refer to `sentry.tracing.dbal.enabled` in this paragraph

Reply:
Good call. I edited the code sample to remove everything but:

```
dbal: # DB queries
      enabled: true
```

Reply:
To avoid any mistake, you should use the full config path in the snippets, otherwise users may misinterpret them and think that they have to configur the `dbal` key at the root node of the file which is not what we mean. TL;DR: I would expect the code snippet to be

```yaml
sentry:
    tracing:
        dbal:
            enabled: true
```